pkg/reconciler/events/cloudevent/cloudeventclient.go

```diff
@@ -33,7 +33,6 @@ func init() {
 	injection.Default.RegisterClient(func(ctx context.Context, _ *rest.Config) context.Context {
 		return withCloudEventClient(ctx)
 	})
-	injection.Dynamic.RegisterDynamicClient(withCloudEventClient)
```

Comment:
Do you know what replaced this?

Reply:
Nothing, it was removed altogether. See https://github.com/knative/pkg/issues/2739#issuecomment-1550452957. It was not really used anywhere.

Reply:
would you mind adding a note about this to the commit message/pr description?

Reply:
Actually, I am already linking it from the commit message (and saying it's not used), is that good enough ?

Reply:
Ah my bad! I normally just look at the PR description and assume the commit message is the same :) Can you squash your commits?

Reply:
@lbernick I can 🙃 I wanted to split it into several part though, one for the bump, one the updating the kubernetes min version and the rest for fixing some changes, but.. I can also make it one big commit 😛